src/spec-converter.js

```diff
@@ -90,9 +90,9 @@ export class SpecConverter {
         var dims = gplSpec.sources['/'].dims;
 
         var reduceIterator = (row, key) => {
-
-            if (_.isObject(row[key]) && !_.isDate(row[key])) {
-                _.each(row[key], (v, k) => (row[key + '.' + k] = v));
+            let rowKey = row[key];
+            if (utils.isObject(rowKey)) {
```

Comment:
Object.keys(new Date()) return [] (empty array) so it does nothing. But I can add it back if you want to be more explicit and safe.